src/platforms/javascript/common/performance.mdx

```diff
@@ -65,6 +65,34 @@ _Example:_
 
 _NOTE:_ You need to make sure your web server CORS is configured to allow the `sentry-trace` header. The configuration might look like `"Access-Control-Allow-Headers: sentry-trace"`, but the configuration depends on your set up. If you do not allow the `sentry-trace` header, the request might be blocked.
 
+### Connecting Backend and Frontend Traces
+
+When instrumenting both frontend and backend with Sentry (Node.js or other
+supported platform), it is possible to connect the pageload transaction on the
+frontend with the trace that originated the page in the backend using a special
+`<meta>` tag in the `<head>` of the HTML document.
+
+Include in the rendered page a meta tag like so:
+
+```html
+<html>
+<head>
+  <meta name="sentry-trace" content="{{ span.toTraceparent() }}">
+  <!-- ... -->
+```
+
+The `content` attribute should have a special string generated by the Sentry
+SDK. The exact syntax depends on your platform and template library.
+
+<Alert level="info" title="Note"><markdown>
+
+In this particular case, the `<meta>` tag is used to propagate tracing metadata,
+replacing the use of the equivalent `sentry-trace` HTTP header described
+earlier. That is because browsers do not provide any API for reading the
+response headers of the current page.
```

Comment:
```suggestion
In this example, the `<meta>` tag is used to propagate tracing metadata,
replacing the use of the equivalent `sentry-trace` HTTP header described
earlier. This is because browsers do not provide any API for reading the
response headers of the current page.
```